src/include/btmem.h

```diff
@@ -339,6 +339,7 @@ struct __wt_page_modify {
                     WT_INSERT *ins; /* Insert list reference */
                     WT_ROW *ripcip; /* Original on-page reference */
                     WT_UPDATE *onpage_upd;
+                    bool restore;
                 } * supd;
                 uint32_t supd_entries;
                 bool supd_restore;
```

Comment:
Can you clarify the difference between supd_restore and restore with a comment around here?